### PR TITLE
feat: update Start with Gravity-UI block description, add npx cra from template

### PR DESCRIPTION
**Pull request by linkouth**
https://github.com/gravity-ui/gravity-ui-cra-template/tree/main

### CODE REVIEW

File: src/content/landing.ts [hunk 1]
```diff
@@ -155,9 +155,8 @@ const typedLanding: CustomPageContent = {
             color: '#23151e',
             image: bannerAsset,
             commands: [
-                'git clone git@github.com:gravity-ui/uikit-example-cra.git my-project && cd my-project',
-                'npm i',
-                'npm run start',
+                'npx create-react-app ./my-project --template gravity-ui && cd my-project',
+                'npm run dev',
```
Comment (imsitnikov) [inline src/content/landing.ts:159]: 1) After installing and "npm run dev" I've seen a blank page.
2) Why should we use `create-react-app` if we use `app-builder` in template and don't use `react-scripts`? It is not actually a CRA application.

**Review by imsitnikov — CHANGES_REQUESTED** [review]

File: src/content/landing.ts (hunk 1 above)
Comment (linkouth) [reply src/content/landing.ts:161]: 1. Hmm, have you opened the app on `http://localhost:8000/`?
2. Good point. I've removed cra call, and will change repo name later, is it ok?

Comment (imsitnikov) [reply src/content/landing.ts:161]: it works on port 8000, but:
Screenshot 2023-09-24 at 14 28 58

**Comment by imsitnikov** [comment]
@linkouth I'll close this PR, we've decided to prepare our install script instead of `create-react-app`.

**Pull request closed without merging**